src/hotspot/share/gc/shenandoah/shenandoahOldGC.cpp

```diff
@@ -157,5 +157,8 @@ bool ShenandoahOldGC::collect(GCCause::Cause cause) {
     LogStream ls(lt);
     result.print_on("Old Mark", &ls);
   }
+
+  entry_reset_after_collect();
```

Comment:
Not sure we want to reset old region bitmaps after old marking is compete. Shenandoah opportunistically uses the bitmap for old regions during remembered set scan (it's faster than walking the heap).

Reply:
The 'reset-after' won't reset bitmaps for old gen, it only reset for young gen, which is skipped in bootstrap cycle